src/hokusai/hokusai.yml

```diff
@@ -1,6 +1,6 @@
 # Orb Version 0.1.0
 
-version: 2.1
+version: 2.2
```

Comment:
This version field is circleci's orb schema version number. Don't change this. Instead, change the _comment_ on line 1.

Reply:
ahhhhhh thanks